django_girls_tutorial/step2/sehwa/djangogirls/blog/api/views.py

```diff
@@ -43,3 +42,31 @@ def retrieve_post_detail(request, id):
             },
             status=200,
         )
+
+
+class CommentListCreate(View):
+    def get(self, request, post_id):
+        try:
+            post = Post.objects.get(pk=post_id)
+        except Post.DoesNotExist:
+            return JsonResponse({"message": "post가 없습니다."}, status=404)
+
+        comments = Comment.objects.filter(post=post, created_date__lte=timezone.now()).order_by("created_date")
+        comment_list = [{"author": comment.author, "text": comment.text} for comment in comments]
+
+        return JsonResponse({"data": comment_list})
```

Comment:
nipick, 

```suggestion
        return JsonResponse({"data": [{"author": comment.author, "text": comment.text} for comment in comments]})
```
변수 선언안해줘도 코드로 인지 가능해서 인라인해서 써줘도 될거 같애~